autoconnect/autoconnect-ws/autoconnect-ws-sm/src/unidentified.rs

```diff
@@ -79,7 +79,19 @@ impl UnidentifiedClient {
             "💬UnidentifiedClient::on_client_msg Hello! uaid: {} existing_user: {}",
             uaid, existing_user,
         );
-        let _ = self.app_state.metrics.incr("ua.command.hello");
+        self.app_state
+            .metrics
+            .incr_with_tags("ua.command.hello")
+            .with_tag("uaid", {
+                if existing_user {
+                    "existing"
+                } else if flags.reset_uaid {
```

Comment:
We want what you had previously for reassigned (comparing the uaids), `reset_uaid` is something else. It's only triggered when that old `record_version` field in the db is bad/invalid, which is probably never at this point.

Technically this flag is a form of a "reassign" but it's delayed: the flag triggers the user to be dropped after it checks/acks all its messages from storage. So I'd argue its "reassign" doesn't occur in this hello'd session w/ the flag set but instead during the following session after the user drop/disconnection.

Have we ever used/bumped `record_version` (it's currently at `1`, was it ever `0` or non-existent?) or has it always been theoretical?

Reply:
IIRC, `record_version` is there in case we ever decided to radically change the record. There was a change we did a long time ago, which may have introduced this field.